### PR TITLE
Added CSS text-anchor property

### PR DESCRIPTION
**Pull request by meyerweb**
### Description

Adds documentation of the CSS property `text-anchor`, which replicates an SVG attribute.

### Motivation

The page currently doesn’t exist, and it needs to be created (see https://github.com/mdn/content/issues/34763).

### CODE REVIEW

File: files/en-us/web/svg/attribute/text-anchor/index.md [hunk 1]
```diff
@@ -92,6 +92,11 @@ svg {
 
 {{Specifications}}
 
+## See Also
```
Comment (Josh-Cena) [inline files/en-us/web/svg/attribute/text-anchor/index.md:95]: This comes after browser compat and "also" should be lowercase

Comment (meyerweb) [reply files/en-us/web/svg/attribute/text-anchor/index.md]: Fixed.

**Review by bsmth — CHANGES_REQUESTED** [review]
Thank you! All good except for the `CSSInfo` issue. Either we omit it or add an entry to `mdn/data`.

**Comment by meyerweb** [comment]
> Thank you! All good except for the `CSSInfo` issue. Either we omit it or add an entry to `mdn/data`.

Per [Josh-Cena’s comment in 35107](https://github.com/mdn/content/pull/35107#issuecomment-2263714975), these should stay even if broken.  (This is the case on several other CSS pages, in fact.)  I can look into adding the information to `mdn/data` when I return from holiday, but in the meantime I believe this can be merged.

**Comment by bsmth** [comment]
> > Thank you! All good except for the `CSSInfo` issue. Either we omit it or add an entry to `mdn/data`.
> 
> Per [Josh-Cena’s comment in 35107](https://github.com/mdn/content/pull/35107#issuecomment-2263714975), these should stay even if broken. (This is the case on several other CSS pages, in fact.) I can look into adding the information to `mdn/data` when I return from holiday, but in the meantime I believe this can be merged.

OK got it. I've opened a tracker in mdn/data so we don't forget!

**Review by bsmth — APPROVED** [review]
Thanks, Eric!

**Pull request merged**